samza-core/src/main/java/org/apache/samza/operators/TableImpl.java

```diff
@@ -20,22 +20,26 @@
 
 import java.io.Serializable;
 import org.apache.samza.table.Table;
-import org.apache.samza.table.TableSpec;
+import org.apache.samza.table.descriptors.TableDescriptor;
 
 
 /**
- * This class is the holder of a {@link TableSpec}
+ * This class is the holder of a table Id
  */
 public class TableImpl implements Table, Serializable {
```

Comment:
Do we still need this class? Looks like this isn't doing much.

Reply:
It is used in building the DAG in fluent API, but replacing it with table Id might have side effects in the future, if we need to add attach any properties to a table. I'd say let's keep it for now.